lib/setup/sink.go

```diff
@@ -0,0 +1,122 @@
+package setup
+
+import (
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/rs/zerolog/log"
+	"github.com/urfave/cli/v2"
+	"math"
+	"net/url"
+	"plane.watch/lib/sink"
+	"plane.watch/lib/tracker"
+	"strconv"
+	"strings"
+)
+
+var (
+	prometheusOutputFrame = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "pw_ingest_output_frame_total",
+		Help: "The total number of raw frames output. (no dedupe)",
+	})
+	prometheusOutputFrameDedupe = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "pw_ingest_output_frame_dedupe_total",
+		Help: "The total number of deduped frames output.",
+	})
+	prometheusOutputPlaneLocation = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "pw_ingest_output_location_update_total",
+		Help: "The total number of plane location events output.",
+	})
+)
```

Comment:
Thinking out aloud... could we export these metrics with a label for the bus type? Would we even have multiple busses linked up?? If not, then don't bother.

Reply:
while it is *possible* i don't think we will in practice